test/api/capi/test_capi_appender.cpp

```diff
@@ -4,6 +4,33 @@
 using namespace duckdb;
 using namespace std;
 
+namespace {
+
+struct CAPIAppender {
+public:
+	CAPIAppender(CAPITester &tester, const char *schema, const char *table) {
+		auto status = duckdb_appender_create(tester.connection, schema, table, &appender);
+		REQUIRE(status == DuckDBSuccess);
+	}
+	~CAPIAppender() {
+		auto status = duckdb_appender_flush(appender);
```

Comment:
`Flush` is not necessary here. `auto status = duckdb_appender_close(appender)` should be sufficient.

Reply:
I looked at a bunch of existing tests and they seem to follow this pattern of flush + close + destroy ?

Reply:
That's misleading, in a way. It's not wrong to do it, but `Close` flushes all data implicitly. The only difference is that `Flush` keeps the appender open, i.e., `Flush` without `Close` leaks memory, but `Close` without `Flush` is perfectly fine. 🤷‍♀️

Reply:
`Flush` makes sense if you want to commit a part of the data occasionally.